### ISSUE TITLE
[Legitimate Site Blocked] shido.market

### ISSUE THREAD

**Issue by bjornbonnevier** [MetaMask/eth-phishing-detect#8911]
https://shido.market/

Hello Metamask,

This NFT Marketplace is falsely flagged. It is powered by NFTIFY - https://nftify.network/

Our website is not blocked by Ethereum Phising detector (https://metamask.github.io/eth-phishing-detect/), nor is it on this list Cryptoscamdb (https://cryptoscamdb.org/search)

Please if you can help us with this matter, we are Shido Finance (https://shido.finance)

Thank you.

Kind regards,
Bjorn Bonnevier

CEO & DEV

Shido Finance

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/8906
